students/mike_newton/Trapezoidal_Integrator/trapz.py

```diff
@@ -0,0 +1,38 @@
+import math
+
+#trapz will be passed the function name, a dictionary of arguments/coefficients required for that function,
+#a starting point (a) and a stopping point (b).  if no arguments or coefficients are requirements,
+#the user will simply pass the word None in the args place when calling trapz.
+
+#each individual function will be responsible for unpacking and using the arguments passed to it from trapz
+
+def trapz(funct, args, a, b):
```

Comment:
Did you mean to use *args here? Here's a good, quick explanation on this: 
http://stackoverflow.com/questions/36901/what-does-double-star-and-star-do-for-python-parameters